.tekton/osc-caa-pull-request.yaml

```
@@ -29,6 +29,8 @@ spec:
     value: cloud-api-adaptor/Dockerfile.openshift
   - name: path-context
     value: src
+  - name: prefetch-input
+    value: '[{"type": "rpm", "path": "./src/cloud-api-adaptor/"}, {"type": "gomod", "path": "./src/cloud-api-adaptor"}]'
```

For later maintenance (smaller diffs) and readability, maybe rewrite the array in expanded multi-line form

```suggestion
value:
  - type: rpm
    path: ./src/cloud-api-adaptor/
  - type: gomod
    path: ./src/cloud-api-adaptor
```

---

Also I'm curious if it is intentional that the path for `rpm` has a trailing `/` but not for `gomod`? This seems consistent across all your cases, but wondering if there is any reason for that.

---

For the trailing / : No it's not intentional. I tend to add a trailing / when I write the path to a folder, just to mark it as such, but apparently I missed it on the RPM side.
And I guess it's consistent across my PRs only because I did copy/paste between projects - I replaced the path without thinking too much about the format :-)

---

@c3d regarding the formatting: can I do that?
In the documentation, they ask to write it in this format, I think it's intentional.
This is a JSON array embedded in the yaml file. I suspect it's passed directly to Hermeto (ex-Cachi2) who does the pre-fetch, not interpreted by anything that reads this yaml file.

---

I have tried, and I get an error at prefetch time: 
```
Error: InvalidInput: 1 validation error for user input
packages -> 1 -> gomod -> path"
  Extra inputs are not permitted
```

I think I will keep it as it is :-)